src/CocktailDetails/CocktailDetails.scss

```diff
@@ -1,10 +1,12 @@
 .cocktail-details-wrapper {
-	height: 100vh;
+	position: absolute;
```

Comment:
I find position absolute tricky - you're css is enviable!

Reply:
Only thing I could find to keep the space between the header & body component separate!